gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemIntegrationTest.java

```diff
@@ -1643,222 +1619,6 @@ public void testFileCreationSetsAttributes() throws IOException {
         .isEqualTo("value1".getBytes(StandardCharsets.UTF_8));
   }
 
-  @Test
-  public void testFileCreationUpdatesParentDirectoryModificationTimestamp()
```

Comment:
I think we need to keep these tests, verify the parent dir mtime is not updated.

Reply:
Because, after this PR, we will not have anything that will update it and this is not out goal to not have them updated (essentially we rely on whatever GCS is doing) - I'm not sure that we should test something that we do not touch.